db/migrate/20180627220531_add_enable_program_outcome_exporter_to_organization.rb

```diff
@@ -0,0 +1,7 @@
+class AddEnableProgramOutcomeExporterToOrganization < ActiveRecord::Migration[5.1]
+  def change
+    change_table :organizations do |t|
+      t.boolean :enable_program_outcome_exporter
```

Comment:
I think we want an export type here... I don't want a new column for each new type we create